docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,9 @@ The following table lists all known issues that are currently active and affecti
 | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------- | ----------------- |
+| [MicroK8s](../integrations/microk8s.md) does not support multi-node control plane clusters. The upgrade strategy, `InPlaceUpgrade`, is the only option available for use.                                                                                                                                                                           | No workaround is available.                                                                                                                                                                                                                                                    | June 1, 2024   | Packs             |
+| Clusters using [MicroK8s](../integrations/microk8s.md) as the Kubernetes distribution, the control plane node fails to upgrade when using the `InPlaceUpgrade` strategy for sequential upgrades, such as upgrading from version 1.25.x to version 1.26.x and then to version 1.27.x.                                                                | Refer to the [Control Plane Node Fails to Upgrade in Sequential MicroK8s Upgrades](../troubleshooting/pack-issues.md) troubleshooting guide for resolution steps.                                                                                                              | June 1, 2024   | Packs             |
+| Clusters usig [MicroK8s](../integrations/microk8s.md) as the Kubernetes distribution, using the `RollingUpgrade` strategy for clusters with less than three control plane nodes may cause the API server to be down during the upgrade, making the cluster inaccessible.                                                                            | A workaround is to deploy clusters with three or more control plane nodes.                                                                                                                                                                                                     | June 1, 2024   | Packs             |
```

Comment:
This one is superseeded by the first known issue as we can't deploy multi-node control plane for now. Multi-node will be supported in a subsequent Palette version, and this specific issue may still exist, so maybe we can just comment it?

Reply:
Removing